.gitignore

```diff
@@ -76,3 +76,6 @@ _configtest.o.d
 sklearn/utils/seq_dataset.pyx
 sklearn/utils/seq_dataset.pxd
 sklearn/linear_model/sag_fast.pyx
+venv/
+sklearn/model_selection/.ipynb_checkpoints/
+examples/compose/.ipynb_checkpoints/
```

Comment:
You are still committing the change in `.gitignore`. You need to revert these and not commit these changes.